keras_nlp/layers/start_end_packer_test.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Start End Packer layer."""
+
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers.start_end_packer import StartEndPacker
+
+
+class StartEndPackerTest(tf.test.TestCase):
+    def test_dense_input(self):
+        input_data = tf.constant([5, 6, 7])
+        start_end_packer = StartEndPacker(sequence_length=5)
+        output = start_end_packer(input_data)
+        expected_output = [5, 6, 7, 0, 0]
+        self.assertAllEqual(output, expected_output)
+
+    def test_dense_input_error(self):
+        input_data = tf.constant([[5, 6, 7]])
+        start_end_packer = StartEndPacker(sequence_length=5)
+        with self.assertRaises(ValueError):
```

Comment:
Please include the error message (just prefix works) to ensure we are capturing the right error.

Reply:
Done! I've reported the whole error message. Hope that's okay?